app/views/case_logs/edit.html.erb

```diff
@@ -1,10 +1,10 @@
 <%= turbo_frame_tag "case_log_form", target: "_top" do %>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds-from-desktop">
-      <h1 class="govuk-heading-xl">Tasklist for log
+      <h1 class="govuk-heading-l">Tasklist for log
         <%= @case_log.id %></h1>
 
-      <h2 class="govuk-heading-s govuk-!-margin-bottom-2">This submission is
+      <h2 class="govuk-heading-m govuk-!-margin-bottom-2">This submission is
```

Comment:
I’m going to be horribly inconsistent here and say that in this particular case, the heading should maintain the small size. Semantically it is a second level heading, but should be slightly deemphasised from the section headings used in the task list below.